semantics.json

```diff
@@ -32,10 +32,10 @@
               {
                 "name": "title",
                 "type": "text",
-                "label": "The title of this tap",
+                "label": "Text displayed when pause",
```

Comment:
"Text displayed when pause" => "Text displayed when the video is paused"
Is that too long?

Reply:
@schup i was mistaken the text should be "Title text displayed on pause" is that better?